packages/react-dom/src/client/ToStringValue.js

```diff
@@ -35,3 +35,52 @@ export function getToStringValue(value: mixed): ToStringValue {
       return '';
   }
 }
+
+/**
+ * Returns true only if Trusted Types are available in global object and the value is a trusted type.
+ *
+ * Trusted Types undergo a change where window.TrustedTypes was renamed to window.trustedTypes
+ * (https://github.com/WICG/trusted-types/pull/205).
+ */
+function isTrustedTypesValue(value: any): boolean {
```

Comment:
Can we make the function declaration itself conditional and check the globals at startup?

```
let isTTValue =
  typeof TT !== "undefined" ?
    impl1 :
    typeof tt !== "undefined" ?
      impl2 :
      impl3
```

Each implementation would assume a specific global exists, and the last one would be a shim that always returns false.

This is to reduce the runtime checks.